semanticcpg/src/main/scala/io/shiftleft/semanticcpg/sarif/SarifConfig.scala

```diff
@@ -0,0 +1,26 @@
+package io.shiftleft.semanticcpg.sarif
+
+import io.shiftleft.semanticcpg.sarif.SarifConfig.SarifVersion
+import io.shiftleft.semanticcpg.sarif.v2_1_0.JoernScanResultToSarifConverter
+import org.json4s.Serializer
+
+import java.net.URI
+
+case class SarifConfig(
+  toolName: String = "Joern",
+  toolFullName: String = "Joern - The Bug Hunter's Workbench",
+  toolInformationUri: URI = URI("https://joern.io"),
+  organization: String = "Joern.io",
+  semanticVersion: String = "4.x.x",
```

Comment:
I see the Joern REPL has a `version` call, which I can use to dynamically load this - one sec...

Reply:
So `semanticVersion` refers to the joern version? 
If you get that from joern via e.g. `io.joern.joerncli.console.JoernConsole.version`, then ocular would also refer to the joern version, which is probably fine. 
That can be changed, but at the same time we should keep things simple... 
just to make sure I understand correctly, can you elaborate on what that version means?

Reply:
This version refers to the tool. I have a branch on `Ocular` which implements this work and runs this configuration before the REPL begins: https://github.com/ShiftLeftSecurity/ocular/compare/master...dave/ocular-sarif#diff-36b4f70cbc3441ca2033a3f58698e662fabe8f4e3fcaae662f7ba3c414782d00R55

I should likely name `organization` and `semanticVersion` more consistently with the `tool` prefix

Reply:
@mpollmeier I see that `joerncli` is not accessible here - where do you suggest I initialize this kind of config value for the REPL? I see there are some `runBefore`-style hooks in the BridgeBase interface but I'm wondering where specifically may be best

Reply:
👍🏻 sounds good